src/main/java/org/fundacionjala/coding/ariel/Kata6.java

```diff
@@ -0,0 +1,39 @@
+package org.fundacionjala.coding.ariel;
+
+import java.util.TreeMap;
+
+/**
+ * Kata6.
+ * by Ariel Gonzales Vargas.
+ */
+public class Kata6 {
+
+    /**
+     * @param numbers An integer array.
+     * @return An changed sortTwisted37 array.
+     */
+    public Integer[] sortTwisted37(Integer[] numbers) {
+        TreeMap<Integer, Integer> series = new TreeMap<Integer, Integer>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)